.github/workflows/create-release-pr.yml

```diff
@@ -10,9 +10,6 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: actions/setup-node@v4
-        with:
-          node-version: 18
```

Comment:
mostly just curious, but why did you remove this?

Reply:
Got a little too aggressive 🗡️